autospec/buildreq.py

```diff
@@ -901,6 +906,27 @@ def scan_for_configure(dirn):
     else:
         config.autoreconf = False
 
+    if buildpattern.default_pattern == "distutils3":
+        # Try and grab the pypi details for the package
+        pypi_api_url = f"https://pypi.python.org/pypi/{tname}/{tversion}/json"
+        pypi_json = download.do_curl(pypi_api_url)
+        if pypi_json:
+            package_pypi = json.loads(pypi_json.getvalue().decode("utf-8"))
+            if package_pypi.get("info"):
+                info = package_pypi["info"]
+                if info.get("name"):
+                    pypi_provides = info["name"]
+                if info.get("requires_dist"):
+                    # Not yet adding requires
+                    pass
+                if info.get("license"):
+                    # The license field is freeform, might be worth adding though
```

Comment:
Maybe print it to the console as a suggestion.

Reply:
That seems reasonable enough to just do, thanks!